hydromt/workflows/grid.py

```diff
@@ -218,12 +226,13 @@ def grid_from_geodataframe(
     rename: Optional[Union[Dict, str]] = dict(),
     all_touched: Optional[bool] = True,
 ) -> xr.Dataset:
-    """
-    Prepares data variable(s) resampled to grid_like object by rasterizing the data from ``gdf``.
-    Several type of rasterization are possible:
-        * "fraction": the fraction of the grid cell covered by the gdf shape is returned.
-        * "area": the area of the grid cell covered by the gdf shape is returned.
-        * "value": the value from the variables columns of gdf are used. If this is used, variables must be specified.
+    """Prepare data variable(s) resampled to grid_like object.
```

Comment:
Same here, good to keep the first line else this will be exactly the same as grid_from_raster_reclass